Unable to upgrade

Running `paru -Syu` produces this error message when trying to upgrade declaro


==> Making package: declaro-git 1.0.0.r17.ga62ee66-2 (Wed 02 Jul 2025 02:44:47 PM EEST)
==> Retrieving sources...
  -> Updating declaro git repo...
==> Validating source files with sha256sums...
    declaro ... Skipped
==> Making package: declaro-git 1.0.0.r17.ga62ee66-2 (Wed 02 Jul 2025 02:44:48 PM EEST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Updating declaro git repo...
==> Validating source files with sha256sums...
    declaro ... Skipped
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Creating working copy of declaro git repo...
Cloning into 'declaro'...
done.
==> Starting pkgver()...
==> Updated version: declaro-git 1.0.3.r9.g2aa1c44-1
==> Sources are ready.
declaro-git-0-3: parsing pkg list...
==> Making package: declaro-git 1.0.3.r9.g2aa1c44-1 (Wed 02 Jul 2025 02:44:50 PM EEST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Starting pkgver()...
==> Removing existing $pkgdir/ directory...
==> Entering fakeroot environment...
==> Starting package()...
Installing declaro...
install -Dm755 src/declaro.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/bin/declaro
install -Dm644 src/commands/clean.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/bin/clean.sh
install -Dm644 src/commands/diff.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/bin/diff.sh
install -Dm644 src/commands/edit.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/bin/edit.sh
install -Dm644 src/commands/generate.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/bin/generate.sh
install -Dm644 src/commands/list.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/bin/list.sh
install -Dm644 src/commands/declare.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/bin/declare.sh
install -Dm644 src/commands/status.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/bin/status.sh
install -Dm644 src/commands/export.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/bin/export.sh
install -Dm644 src/commands/import.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/bin/import.sh
install -Dm644 src/commands/install-config.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/bin/install-config.sh
install -Dm644 src/utils.sh /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/bin/utils.sh
install -d /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/config
cp config/* /home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/usr/share/declaro/config
Done.
Installing configuration...
SUDO=" " ETC_DECLARO_DIR=/home/me/.cache/paru/clone/declaro-git/pkg/declaro-git/etc/declaro IS_CALLED_AS_SUBCOMMAND="false" bash src/commands/install_config.sh
bash: src/commands/install_config.sh: No such file or directory
make: *** [Makefile:35: install-config] Error 127
==> ERROR: A failure occurred in package().
    Aborting...
error: failed to build 'declaro-git-0-3': 
error: packages failed to build: declaro-git-0-3

--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,7 @@ install:
 
 install-config:
 	@echo "Installing configuration..."
-	SUDO="$(SUDO) " ETC_DECLARO_DIR=$(DESTDIR)$(ETC_DECLARO_DIR) IS_CALLED_AS_SUBCOMMAND="false" bash src/commands/install_config.sh
+	SUDO="$(SUDO) " SHRBINDIR="src" ETC_DECLARO_DIR=$(DESTDIR)$(ETC_DECLARO_DIR) IS_CALLED_AS_SUBCOMMAND="false" bash src/commands/install-config.sh
 	@echo "Done."
 
 

--- a/src/commands/install-config.sh
+++ b/src/commands/install-config.sh
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-SHRBINDIR=$(dirname $BASH_SOURCE)
+# HOTFIX: install.config can be called from make and must be path must be adjusted
+SHRBINDIR=${SHRBINDIR:-"$(dirname $BASH_SOURCE)"}
 source $SHRBINDIR/utils.sh
 
 #this command is the only declaro subcommand that can also be called from Makefile

